Extensions/Multiplayer/multiplayertools.ts

```diff
@@ -646,6 +650,37 @@ namespace gdjs {
       );
     };
 
+    const sendHeartbeatToBackend = async function () {
+      const gameId = gdjs.projectData.properties.projectUuid;
+      const playerId = gdjs.playerAuthentication.getUserId();
+      const playerToken = gdjs.playerAuthentication.getUserToken();
+
+      if (!gameId || !playerId || !playerToken || !_lobbyId) {
+        logger.error(
+          'Cannot keep the lobby playing without the game ID or player ID.'
+        );
+        return;
+      }
+
+      const rootApi = isUsingGDevelopDevelopmentEnvironment
+        ? 'https://api-dev.gdevelop.io'
+        : 'https://api.gdevelop.io';
+      const headers = {
+        'Content-Type': 'application/json',
+      };
+      let heartbeatUrl = `${rootApi}/play/game/${gameId}/public-lobby/${_lobbyId}/action/heartbeat`;
+      headers['Authorization'] = `player-game-token ${playerToken}`;
+      heartbeatUrl += `?playerId=${playerId}`;
+      const players = gdjs.multiplayerMessageManager.getConnectedPlayers();
+      await fetch(heartbeatUrl, {
```

Comment:
Should this be try/catched to ignore the error (an error during an heartbeat is probably something we can ignore once) or retried, or retried and then ignored (probably would be the best solution)?

Reply:
good idea, will retry once